app/services/manage_courses_api.rb

```diff
@@ -0,0 +1,36 @@
+module ManageCoursesAPI
+  class Connection
+    BASE_URL = Settings.manage_api.base_url
+    TOKEN = Settings.manage_api.secret
+
+    def self.api
+      Faraday.new(url: BASE_URL) do |faraday|
+        faraday.response :logger unless Rails.env.test?
+        faraday.adapter Faraday.default_adapter
+        faraday.headers['Content-Type'] = 'application/json'
+        faraday.headers['Authorization'] = "Bearer #{TOKEN}"
+      end
+    end
+  end
+
+  class Request
+    class << self
+      def publish_course(email, provider_code, course_code)
+        response = api.post("/api/Publish/internal/course/#{provider_code}/#{course_code}", email: email)
+        begin
+          if response.status == 200
+            JSON.parse(response.body)["result"]
+          else
+            false
+          end
+        rescue StandardError => e
+          raise StandardError("Could not publish course #{response.status}, #{response.body}, #{e}")
```

Comment:
Maybe 
`provider_code` 
`course_code`

as well

Reply:
May a test for this part as well since its system to system integration as well

Reply:
Done and done